Makefile

```diff
@@ -43,7 +43,7 @@ TAIL_COUNT ?= 10
 
 LINKFLAGS := -g
 LIBS := -lz
-CXXFLAGS := -g -Wall
+CXXFLAGS := -g -Wall -Wno-unqualified-std-cast-call
```

Comment:
Seems extraneous - there are a lot of other warnings that need cleaning up.

Reply:
Would it be okay, if I just replace all ~50 instances of `move()` with `::std::move()` ?